Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -336,40 +326,36 @@ open class NavigationEventsManager {
     }
     
     /**
-     Send feedback about the current road segment/maneuver to the Mapbox data team.
+     Create feedback about the current road segment/maneuver to be sent to the Mapbox data team.
```

Comment:
Maybe this documentation should mention `NavigationViewController.detailedFeedbackEnabled` and the effect it has (or doesn’t have) on this custom deferred feedback flow.

Reply:
I'm not sure that mentioning `MapboxNavigation` classes in the Core module is a good idea. 🤔